datadog_lambda/trigger.py

```diff
@@ -115,9 +115,9 @@ def parse_event_source(event: dict) -> _EventSource:
     event_source = None
 
     # Get requestContext safely and ensure it's a dictionary
-    request_context = event.get("requestContext", {})
+    request_context = event.get("requestContext") or {}
     if not isinstance(request_context, dict):
-        request_context = {}
+        request_context = None
```

Comment:
Teeeeeechnically, in this situation you could just leave `event.get("requestContext")` without the `or {}`. Because you're properly handling the None case below.